tml/datafile.py

```diff
@@ -107,10 +107,6 @@ def __init__(self, map_path):
                     license = decompress(self.get_compressed_data(f, license))[:-1]
                 else:
                     license = None
-                if settings > -1:
-                    settings = decompress(self.get_compressed_data(f, settings)).split(b'\x00')[:-1]
-                else:
-                    settings = None
```

Comment:
Why those changes?

Reply:
I don't know, but
- DDnet removed it from their fork
- With teeworlds 0.7.2 maps it's doesn't work (if those lines are still there)